test/integration/short/metadata-tests.js

```diff
@@ -1786,7 +1788,8 @@ describe("metadata @SERVER_API", function () {
                     );
                 },
             );
-            vit(
+            // Potentially delete as it requires datastax DB?
+            /* vit(
                 "dse-6",
```

Comment:
This requires more investigation, because ScyllaDB supports CDC. Let's make an issue for that and for name have this test disabled.